code/src/components/GamePlay.js

```diff
@@ -0,0 +1,217 @@
+/* eslint-disable linebreak-style */
+import React from 'react';
+import styled from 'styled-components/macro';
+import { useSelector, useDispatch } from 'react-redux';
+import { nextMove, game } from 'reducers/game';
+import uniqid from 'uniqid';
+import { Text } from './StartingPage.js';
+import FirstImage from '../assets/1-0.jpg'
+import SecondImage from '../assets/1-1.jpg'
+import ThirdImage from '../assets/0-1.jpg'
+import FourthImage from '../assets/0-2.jpg'
+import FifthImage from '../assets/0-3.jpg'
+import SixthImage from '../assets/1-3.jpg'
+import DefaultImg from '../assets/default.jpg'
+
+export const GamePlay = () => {
+  const story = useSelector((store) => store.game.position.description)
+  console.log(story)
+  const dispatch = useDispatch()
+
+  const onButtonClick = (direction) => {
+    dispatch(nextMove(direction))
+  }
+
+  const onRestartButton = () => {
+    dispatch(game.actions.restartGame())
+  }
+
+  const direction = useSelector((store) => store.game.position.actions)
+  console.log(direction)
+  const noActionsLeft = direction.length === 0
+  console.log(noActionsLeft)
+  const currentCoordinate = useSelector((store) => store.game.position.coordinates)
+  console.log(currentCoordinate)
+
+  let backgroundImage;
+  switch (currentCoordinate) {
+    case '1,0':
+      backgroundImage = FirstImage;
+      break;
+    case '1,1':
+      backgroundImage = SecondImage;
+      break;
+    case '0,1':
+      backgroundImage = ThirdImage;
+      break;
+    case '0,2':
+      backgroundImage = FourthImage;
+      break;
+    case '0,3':
+      backgroundImage = FifthImage;
+      break;
+    case '1,3':
+      backgroundImage = SixthImage;
+      break;
+    default:
+      backgroundImage = DefaultImg;
```

Comment:
Nice way to switch backgrounds, another way to do it is to have an object with all the background images as values, with the coordinates as keys.   
`const backgroundImages = {
  '0,0': '/assets/img2.jpg',
  '1,0': '/assets/img3.jpg',
  '1,1': 'assets/img4.jpg',
  '0,1': 'assets/img5.jpg',
  '0,2': 'assets/img6.jpg',
  '0,3': 'assets/img7.jpg',
  '1,3': 'assets/img8.jpg'
};`